docs/src/main/sphinx/connector/hive.rst

```diff
@@ -483,6 +483,11 @@ with Parquet files performed by the Hive connector.
         definition. The equivalent catalog session property is
         ``parquet_use_column_names``.
       - ``true``
+    * - ``parquet.optimized-writer.validation-percentage``
+      - Percentage of parquet files to validate after write by re-reading the whole file
+        when ``parquet.experimental-optimized-writer.enabled`` is set to ``true``.
+        The equivalent catalog session property is ``parquet_optimized_writer_validation_percentage``.
```

Comment:
Why percentage instead of just a fraction?

Reply:
It's percentage to be consistent with existing orc_writer_validate_percentage flag